test/settings.py

```diff
@@ -98,11 +98,10 @@
 EMAIL_HOST_USER = os.environ.get('EMAIL_HOST_USER', 'user@domain')
 DEFAULT_FROM_EMAIL = EMAIL_HOST_USER
 EMAIL_HOST_PASSWORD = os.environ.get('EMAIL_HOST_PASSWORD', 'password')
-EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
```

Comment:
`console` backend seems apt for testing to me. Is this really an intended change?

Reply:
Yes. I wanted to test an actual email. I will add it to environment variables for eveyone preference.